build.gradle

```diff
@@ -130,15 +131,19 @@ test {
     systemProperty('project.root', project.rootDir.absolutePath)
 }
 
-integTestRunner {
+testClusters.integTest {
+    testDistribution = 'oss'
+}
+
+integTest.runner {
     // add "-Dtests.security.manager=false" to VM options if you want to run integ tests in IntelliJ
     systemProperty 'tests.security.manager', 'false'
     // allows integration test classes to access test resource from project root path
     systemProperty('project.root', project.rootDir.absolutePath)
 
     // Tell the test JVM if the cluster JVM is running under a debugger so that tests can use longer timeouts for
     // requests. The 'doFirst' delays reading the debug setting on the cluster till execution time.
-    doFirst { systemProperty 'cluster.debug', integTestCluster.debug }
+    // doFirst { systemProperty 'cluster.debug', integTestCluster.debug }
```

Comment:
what was the function of this line?

Reply:
This line is commented out because of the removal of integTestCluster in new ES. Will update after we fixed Jacoco integration. I'll address this change in #376 for reference too.